RevenueCatUI/PaywallView.swift

```diff
@@ -111,15 +104,8 @@ public struct PaywallView: View {
 
     // @PublicForExternalTesting
     init(configuration: PaywallViewConfiguration, paywallViewOwnsPurchaseHandler: Bool = true) {
```

Comment:
I think this `paywallViewOwnsPurchaseHandler` can be removed since it's not being checked anymore?